floodviz/thumbnail/thumbnail.js

```diff
@@ -44,19 +51,17 @@ jsdom.env(
 	[
 		'./floodviz/static/bower_components/d3/d3.js',
 		'./floodviz/static/bower_components/proj4/dist/proj4.js',
```

Comment:
We don't need proj4 for the hydrograph, so this can be removed. Also remove the map div since we are only doing the hydrograph (that's above this on line 47).